readme.md

```diff
@@ -1424,13 +1424,20 @@ your `vite.config.js`.
 
 #### WMR
 
-~~[WMR](https://github.com/preactjs/wmr) supports [Rollup][] plugins directly by
-[pushing them to `config.plugins`](https://github.com/preactjs/wmr/tree/main/packages/wmr#configuration-and-plugins)
-in `wmr.config.js`.~~
+[WMR](https://github.com/preactjs/wmr) supports [Rollup][] plugins directly by
+[adding them to `plugins`](https://wmr.dev/docs/plugins)
+in `wmr.config.mjs`.
 
-WMR *says* it supports Rollup plugin but apparently it fails on `.mdx` files
-before the Rollup plugin.
-See [GH-62](https://github.com/wooorm/xdm/issues/62).
+```js
+import { defineConfig } from 'wmr';
+import xdm from 'xdm/rollup.js';
+
+export default defineConfig({
+   plugins: [
+       xdm({ /* Options... */ })
+   ]
+});
+```
```

Comment:
````suggestion
```js
import {defineConfig} from 'wmr'
import xdm from 'xdm/rollup.js'

export default defineConfig({
  plugins: [
    xdm({/* Options… */})
  ]
})
```
````

Formatted matching this project’s style.

Reply:
Whoops, thanks for catching this.